products/idn/tools/cli/connectors.md

```diff
@@ -0,0 +1,160 @@
+---
+id: cli-connectors
+title: CLI Connectors
+pagination_label: CLI Connectors
+sidebar_label: CLI Connectors
+sidebar_position: 2
+sidebar_class_name: cli-connectors
+keywords: ['cli', 'cli connectors', 'connectors']
+description: Learn about the CLI commands you can use to create, manage, and test connectors in this guide. 
+slug: /tools/cli/connectors
+tags: ['CLI']
+---
+
+## Connectors
+
+Learn about the CLI commands you can use to create, manage, and test connectors in this guide. 
+
+The `connectors` command is a CLI interface for the SaaS Connectivity platform. The CLI is the best way to create, manage, and test SaaS connectors within your tenant. For more information about SaaS Connectivity, refer to the [SaaS Connectivity](https://developer.sailpoint.com/idn/docs/saas-connectivity) guide. 
+
+## Commands 
+
+To create, manage, and test SaaS connectors with the CLI, you can use these commands: 
+
+- [Init project](#init-project)
+- [Create connector](#create-connector)
+- [Upload connector](#upload-connector)
+- [Invoke command](#invoke-command)
+- [List connectors](#list-connectors)
+- [Update connector](#update-connector)
+- [Delete connector](#delete-connector)
+- [Manage tags](#manage-tags)
+- [Get logs](#get-logs)
```

Comment:
We are missing the sail conn validate command here. The validate command runs integration tests against your connector to ensure all the basic functionality works. You can pass in a -r to run it as read only or do a full suite of read write tests.